### DISCUSSION TITLE
为啥我发电了也没发快照给我啊，也没法取消人工回复

### DISCUSSION THREAD

**Discussion by Cherrieslove** [Meloong-Git/PCL#1867] [提问] [answered]
脑溢血急救中心的办法也用了，但是私信还是没动静
![image](https://github.com/Hex-Dragon/PCL2/assets/121227893/abd76e9a-6037-4266-ab8d-a1bd12ff64fa)

![image](https://github.com/Hex-Dragon/PCL2/assets/121227893/0bf84da9-9663-4d7c-955e-2899f9492780)

**Comment by Chiloven945** [comment]
回复下载看看

**Reply by Cherrieslove** [reply]
解决了，谢谢啦

**Comment by iPodtouch6557** [comment, marked as answer]
如果你等了好几十分钟还没回复，那就有问题了
一般延时个十几二十秒正常的，除非你的网络~有点逝~，或者伯特出了点问题……

**Reply by Cherrieslove** [reply]
解决了，方法有用，谢谢

**Comment by WForst-Breeze** [comment]
不是输入识别码三个汉字，而是输入你的启动器的一串识别码

**Reply by Cherrieslove** [reply]
谢谢啦，已经解决了
